### ISSUE TITLE
Optmize mmpt.test.ts

### ISSUE THREAD

**Issue by bgins** [fission-suite/webnative#221]
The `src/fs/protocol/private/mmpt.test.ts` takes 2 seconds or more to run and sometimes causes Jest to time out.

Some performance testing shows that the bottleneck is concurrently adding examples to the mmpt:

https://github.com/fission-suite/webnative/blob/b146c74f733f68e11c07cc79b70fd795eaf825e0/src/fs/protocol/private/mmpt.test.ts#L61-L62

The total time for the "can handle concurrent adds" test and time to add entries were tested. Total time was collected by jest and the time to add entries was collected as follows:
```
const start = performance.now()
await Promise.all(entries.map(entry => mmpt.add(entry.name, entry.cid)))
const stop = performance.now()
console.log((stop - start) / 1000)
```

| Entries | Time to add entries (s) | Total test time (s) |
|---------|-------------------------|---------------------|
| 100     | 0.9115                  | 5.576               |
| 200     | 3.051                   | 8.058               |
| 500     | 9.973                   | 15.442              |
| 1000    | 17.587                  | 23.593              |
| 2000    | 43.334                  | 50.017              |
| 4000    | 91.508                  | 110.896             |

Only one trial was run per entry count, and these results should be considered anecdotal, but they are enough to show us the bottleneck!

After some discussion with @matheus23, we decided that we can lower the number of entries used in this test to 500 or 1000 and still get reasonable coverage.

**Comment by expede** [comment]
@bgins just to check, the "Time to add entries (s)" is for the complete set, yes? (Not per item.)

**Comment by bgins** [comment]
@expede Yep, that's right. The `entries` that are added are created with this code:

https://github.com/fission-suite/webnative/blob/b146c74f733f68e11c07cc79b70fd795eaf825e0/src/fs/protocol/private/mmpt.test.ts#L25-L38

**Comment by expede** [comment]
I threw this into a quick graph

<img width="744" alt="Screen Shot 2021-05-06 at 11 05 47 AM" src="https://user-images.githubusercontent.com/1052016/117345141-0ab7b280-ae5b-11eb-97d5-6c944bc67366.png">

I this nonscientific test, it looks like we pay some concurrency overhead, but less than I expected!
